vast.conf

```diff
@@ -132,7 +125,11 @@ import {
   ; Block until the importer forwarded all data.
   ;blocking = false
 
-  ; Select the table slice type.
+  ; Number of events to be batched in a table slice (this is a target value that
+  ; can be underrun if the source has a low rate).
+  ;table-slice-size = 100
+
+  ; The table slice type (default|arrow).
```

Comment:
This comment needs an update once the renaming PR is merged.

Reply:
It already is, I missed this during the rebase actually. Good catch!